packages/block/test/testdata/goerliCommon.ts

```diff
@@ -74,17 +74,12 @@ export const Goerli: ChainConfig = {
       forkHash: '0xb8c6299d',
     },
     {
-      // The forkHash will remain same as mergeForkIdTransition is post merge,
+      // The forkHash will remain same as MergeNetsplitBlock is post merge,
```

Comment:
This does not look correct (anymore)? Should be GrayGlacier?

Reply:
I'm not tracking your question here?  If I revert the one change (and add the `MergeForkIdTransition` back with null values and just rename to `MergeNetsplitBlock`) the tests still pass as before.

Reply:
Ah it's in test data I see (could not see that on the mobile UI). My point was that there is no `MergeNetsplitBlock` in this common so this comment does not make much sense. It's fine though, it's in the test data.